db_test.go

```diff
@@ -113,14 +113,14 @@ func TestSave(t *testing.T) {
 	for _, e := range examples {
 		Save(e.m)
 	}
-	if Count(TestStruct{}, "") != len(examples) {
+	if Count(TestStruct{}, "") != int64(len(examples)) {
```

Comment:
no need to cast it in here if we do not change the signature of Count to `int64`